source/overloading.tex

```diff
@@ -624,7 +623,7 @@
 public:
   C(int);
 };
-T a = 1;            // ill-formed: \tcode{T(C(1))} not tried
+T a = 1;            // error: \tcode{T(C(1))} not tried
```

Comment:
// error: no viable conversion (`T(C(1))` not considered)

Reply:
Fixed.